lib/beaker/host/windows.rb

```diff
@@ -50,12 +50,17 @@ def self.foss_defaults
         #let's just add both potential bin dirs to the path, include ruby too for `gem`, `ruby`, etc
         'puppetbindir'      => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin',
         'hierabindir'       => '/opt/puppet-git-repos/hiera/bin',
-        'pathseparator'     => ':',
+        'pathseparator'     => ';',
       })
     end
 
     def self.aio_defaults
-      self.foss_defaults
+      h = self.foss_defaults
+      h['puppetvardir']   = '`cygpath -smF 35`/PuppetLabs/puppet/cache'
```

Comment:
OK - a couple quick notes.  Essentially, 2 main categories of things are incorrect:

* The `codedir` should be updated inline with the spec - https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md#puppet-agent-windows

* The `foss_defaults` are using Git style locations for hiera config values, even though AIO installs from the MSI.  I believe these can be omitted.

```ruby
h['puppetcodedir']     = '`cygpath -smF 35`/PuppetLabs/code'

['hieralibdir', 'hierapuppetlibdir', 'hierabindir'].each { |k| h.delete(k) }
```

Since this would now start trashing things from an existing hash, I think it's probably better to just start with a fresh hash, and explicitly define all the values like is done for `pe_defaults` and `foss_defaults`